library/core/src/ptr/non_null.rs

```diff
@@ -502,6 +502,13 @@ impl<T: ?Sized> NonNull<T> {
     #[cfg_attr(miri, track_caller)] // even without panics, this helps for Miri backtraces
     #[stable(feature = "non_null_convenience", since = "1.80.0")]
     #[rustc_const_stable(feature = "non_null_convenience", since = "1.80.0")]
+    #[kani::requires(
+        count <= isize::MAX &&
```

Comment:
This conjunct is a no-op since `count` is an `isize` so it cannot exceed `isize::MAX`.